### DISCUSSION TITLE
Page the Elephant 🐘 [Pages Mini Check-in]

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/community#39354] [Other Feature Feedback, Questions, & Ideas]
We’re midway through November and I’m here to share the latest and greatest in the Pages category. 

📄 **Pros in Pages**: A round of applause for these community citizens 👏
* @dhrubasaha08: For keeping their answer simple to solve a [blank screen for a custom Pages domain](https://github.com/orgs/community/discussions/38990). 
* @neildobson-au: For self-solving why a branch wasn’t deploying. Appreciate you coming back to your [post](https://github.com/orgs/community/discussions/39054) to let us know how you fixed it! 

💡**Inspiration Station**: Fresh content to give you new ideas! 
* **Pages Gem Project**: Have you checked out this [simple Ruby Gem](https://github.com/github/pages-gem) to bootstrap dependencies for setting up and maintaining a local Jekyll environment in sync with GitHub Pages? 

🧹**Housekeeping**: Community on Discussions updates! 
* **Sign up for GitHub’s Customer Research Panel**: You’ll have the opportunity to give feedback on early designs, get access to betas, and inform our product direction ▶️. [Learn more](https://github.com/orgs/community/discussions/386960). 

Have a hero 🦸🏼‍♀️  you want to kudos? Join me in the comments by sharing a community member who has helped you recently!

**Comment by dhrubasaha08** [comment]
Thank you so much for the feedback, I really appreciate it :) @queenofcorgis

**Reply by queenofcorgis** [reply]
You're welcome @dhrubasaha08 ❤️ Thanks for helping answer!

**Comment by spenserblack** [comment]
> * [Learn more](https://github.com/orgs/community/discussions/386960.

I think there's an extra `0` at the end (let go of shift too early? :laughing:). Should it be https://github.com/orgs/community/discussions/38696?

**Reply by queenofcorgis** [reply]
@queenofcorgis now you know a real human wrote this - nice catch, thank you!
